moveit_core/kinematic_constraints/src/kinematic_constraint.cpp

```diff
@@ -655,41 +668,60 @@ ConstraintEvaluationResult OrientationConstraint::decide(const moveit::core::Rob
   if (!link_model_)
     return ConstraintEvaluationResult(true, 0.0);
 
-  std::tuple<Eigen::Vector3d, bool> euler_angles_error;
+  Eigen::Vector3d xyz;
+  Eigen::Isometry3d diff;
```

Comment:
Please use a more descriptive name (e.g. `rotation`).

Also, could these variables all be declared together?

Reply:
I agree that `diff` is not a good name here, but it is used in many other places and not specific to this PR, so maybe do this in a separate PR? Or just change diff in this specific function first and the rest for another PR?

My thinking was that by declaring `euler_angles_error` as close as possible to where it is used, this could make the code easier to follow. Do you think it has the opposite effect?

Reply:
Actually I thought I rewrote this comment, but I must have forgotten to enter it. I don't have a strong opinion on this, we can also leave it be. What about using `rotation_xyz` instead of `xyz` though?

Reply:
> declaring euler_angles_error as close as possible to where it is used

Hmm, this means `xyz` should also be declared later on.

Completely agree that `rotation_xyz` is better than `xyz`, for some reason I thought for a moment it was a position vector...which supports the argument for the new name :)

I'll fix it.

Reply:
I was thinking to also rename `diff` here because it seems a bit silly to be in a separate PR.
But trying a few different longer names such as `diff_transform` I think it only adds redundant information. For me at least, it seems that it is always clear from the context that this is a transform.

I gladly change it if you have better naming ideas. Otherwise, I think this addressed the last comment of you're review.